dox/doxygen/MainPage.dox.in

```diff
@@ -25,6 +25,6 @@
  *
  * \section manual Manual
  *
- * <p>We have a manual! Download it <a href="../download/elastix-@ELASTIX_VERSION@-manual.pdf" onClick="javascript: pageTracker._trackPageview('../download/elastix-@ELASTIX_VERSION@-manual');">here</a>.</p>
+ * <p>We have a manual! Download it <a href="https://github.com/SuperElastix/elastix/releases/download/@ELASTIX_VERSION@/elastix-@ELASTIX_VERSION@-manual.pdf" onClick="javascript: pageTracker._trackPageview('https://github.com/SuperElastix/elastix/releases/download/@ELASTIX_VERSION@/elastix-@ELASTIX_VERSION@-manual');">here</a>.</p>
```

Comment:
Do I understand correctly that `pageTracker._trackPageview(...)` should always have the same URL as  `href=`..., except for the file extension, ".pdf"?

Reply:
@mwjhhuijberts FYI This is the `pageTracker._trackPageview` thing that I meant. No problem to me, but not sure if it's still useful... 🤷

Reply:
not useful anymore. the whole onClick thing can go...